packages/synapse-react-client/src/style/components/_download-cart-page.scss

```diff
@@ -26,35 +35,51 @@
     height: 70px;
     align-items: center;
     list-style: none;
+
     .active {
       padding-top: 5px;
       border-bottom: 5px solid;
       border-bottom-color: SRC.$secondary-action-color;
+
       button {
         color: #515359;
       }
     }
+
     .nav-item {
       font-weight: bold;
       height: 50px;
       width: 290px;
       text-transform: uppercase;
       text-align: center;
       margin-top: 4px;
+
       button {
         font-size: 18px;
         border-width: 0;
         //increase click area
         width: 100%;
         height: 100%;
         padding-bottom: 10px;
+
         .fileCount {
           font-weight: 300;
           margin-left: 5px;
         }
       }
     }
+
+    // make design adjustments for mobile
+    @media (max-width: 768px) {
+      height: auto;
+      flex-direction: column;
+
+      .nav-item button {
+        font-size: 15px;
+      }
+    }
   }
+
   .availableForDownloadTableContainer {
```

Comment:
Would you do anything to the styling here?

To me it would look nicer to center the filtering drop down

```
    @media (max-width: 768px) {
      .filterFilesContainer {
        display: flex;
        justify-content: center;
      }
    }
```

What do you think @nickgros

Reply:
Sounds like a good idea to me! I haven't looked at this too closely but it seems like we don't necessarily need the grid. If it makes sense, you may be able to simplify this if you remove the grid and replace it with  `flex` and set `justify-content: flex-end` outside of the media query, then just override the `justify-content` in your media query.

If you think it looks better, you could further enhance this so the dropdown button takes up the full width, e.g.:

```scss
.filterFilesContainer {
  justify-content: center;
  gap: 16px;
  .dropdown {
    flex: 1;
    .dropdown-toggle {
      width: 100%
    }
  }
}
```

Reply:
We also have this ticket: https://sagebionetworks.jira.com/browse/SWC-6543 - not trying to scope-creep but if you're digging deep into this layout, you could consider also fixing it here.

Instead of going to flex, you could see if it makes sense to keep the grid, and tweak the CSS it to fix this bug at wider screen sizes and get your preferred layout at narrow widths -- up to you.

Reply:
I'm going to call this ticket done and open a separate PR for SWC-6543